<launch>

  <arg name="robot_name" default="$(env ROBOT_NAME)"/>
    <!--- Run AMCL --> 
    <!--include file="$(find amcl)/examples/amcl_omni.launch" /-->

   <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find guide_control)/navigation/costmap_common_params.yaml" command="load" ns="global_costmap" /> 
    <rosparam file="$(find guide_control)/navigation/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find guide_control)/navigation/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find guide_control)/navigation/global_costmap_params.yaml" command="load" /> 
    <rosparam file="$(find guide_control)/navigation/base_local_planner_params.yaml" command="load" />
    <remap from="cmd_vel" to="cmd_vel_dwb"/>
    <remap from="odom" to="$(arg robot_name)/odom"/>
 </node>

</launch> 
